### DISCUSSION TITLE
I have tried multiple times to make ONE item, and I can't.

### DISCUSSION THREAD

**Discussion by MagicalFrog7750** [bridge-core/bridge.#251] [Q&A] [answered]
I've tried for three days straight to make ONE item so I knew how to do it so I could copy and paste it, but the name and texture never worked. The picture is in the files, I even followed a tutorial, but to no avail. The tutorial was made in June 2019 no doubt, but I've tried countless times by myself, and now I've resorted to GitHub, because I have no idea why it's not working.

**Comment by solvedDev** [comment]
GitHub's not the right place for addon help. Please join the bridge. Discord server and use one of the #addon-help channels.
You can also use a preset which automatically creates a fully working item for you.

**Comment by MagicalFrog7750** [comment]
If you don't mind me asking, what's a present and where and how do I use one?

**Comment by solvedDev** [comment, marked as answer]
bridge. toolbar > Tools > Presets

**Comment by solvedDev** [comment]
A preset is a Minecraft feature (item, block, entity,...) that can be created with a couple of clicks. After initiating a preset, the new feature already works inside of Minecraft just fine.
